drivers/bridge/bridge.go

```diff
@@ -147,6 +147,27 @@ func (c *networkConfiguration) Validate() error {
 	return nil
 }
 
+func (c *networkConfiguration) copyDefaultConfigs(option options.Generic, config *networkConfiguration) {
+	if _, ok := option["Mtu"]; !ok {
+		config.Mtu = c.Mtu
+	}
+	if _, ok := option["EnableIPTables"]; !ok {
+		config.EnableIPTables = c.EnableIPTables
+	}
+	if _, ok := option["EnableIPMasq"]; !ok {
+		config.EnableIPMasquerade = c.EnableIPMasquerade
+	}
+	if _, ok := option["EnableICC"]; !ok {
+		config.EnableICC = c.EnableICC
+	}
+	if _, ok := option["EnableUserlandProxy"]; !ok {
+		config.EnableUserlandProxy = c.EnableUserlandProxy
+	}
+	if _, ok := option["EnableIPv6"]; !ok {
+		config.EnableIPv6 = c.EnableIPv6
```

Comment:
Assuming the UI client sets FixedCIDRv6 and default gw v6 but not EnableIPv6 (because now no longer required), the Validate() check about the v6 parameters will not be performed.